calculateMetrics/metrics.py

```diff
@@ -0,0 +1,381 @@
+            added_scores.append(4)
+            username_accesser += 1
+        # This if statment will end if the user was within 1.5 of the standard deviation.
+        elif github_data[username_accesser][2] <= np.average(added_list) + (
+            added_sd * 1.5
+        ) and github_data[username_accesser][2] >= np.average(added_list) - (
+            added_sd * 1.5
+        ):
+            print("appending 3 to added_scores list")
+            added_scores.append(3)
+            username_accesser += 1
+        # This if statment will end if the user was within 2.0 of the standard deviation.
+        elif github_data[username_accesser][2] <= np.average(added_list) + (
+            added_sd * 2.0
+        ) and github_data[username_accesser][2] >= np.average(added_list) - (
+            added_sd * 2.0
+        ):
+            print("appending 2 to added_scores list")
+            added_scores.append(2)
+            username_accesser += 1
+        # This if statment will end if the user was within 2.25 of the standard deviation.
+        elif github_data[username_accesser][2] <= np.average(added_list) + (
+            added_sd * 2.25
+        ) and github_data[username_accesser][2] >= np.average(added_list) - (
+            added_sd * 2.25
+        ):
+            print("appending 1 to added_scores list")
+            added_scores.append(1)
+            username_accesser += 1
+        else:
+            print("Appending 0 to added_scores list")
+            added_scores.append(0)
+            username_accesser += 1
+    global added_overall_score
+    added_overall_score = np.average(added_scores)
+    # These print statments print out the scores for the amount of code the team,
+    # added
+    print("")
+    print("Each users scores are listed below: ")
+    print(added_scores)
+    print("The average of these scores is: ", np.average(added_scores))
+    print(
+        "The team earned a score of: [",
+        np.average(added_scores),
+        "/ 5] for teamwork on lines added.",
+    )
+    print("")
+
+
+def removed_calculator():
+    """This will determine how well the team worked together by analyzing the spread of lines of code removed"""
+    global standard_deviations_list
+    global removed_list
+    print("Lines removed average is: ", np.average(removed_list))
+    print("Lines removed standard deviation is: ", np.std(removed_list))
+    # Standard deviation of lines removed
+    removed_sd = standard_deviations_list[2]
+    removed_scores = []
+    username_accesser = 0
+    list_length = len(github_data)
+    for ef in range(len(github_data)):
+        print("Checking GitHub user: ", github_data[username_accesser][0])
+        # This if/else calculates the amount of code removed to the,
+        # standard deviation in order to grade the team.
+        #Ths if statment will end if the user was within 0.5 of the standard deviation.
+        if github_data[username_accesser][3] <= np.average(removed_list) + (
```

Comment:
Same here with the use of `github_data[username_accesser][3]` and `np.average(added_list)`